ui/index.d.ts

```diff
@@ -66,6 +66,11 @@ export interface IAzureNode<T extends IAzureTreeItem = IAzureTreeItem> {
      * This method combines the environment.portalLink and IAzureTreeItem.id to open the resource in the portal. Optionally, an id can be passed to manually open nodes that may not be in the explorer.
      */
     openInPortal(id?: string): void;
+
+    /**
+     * Displays a 'Loading...' icon and temporarily changes the node's description while `callback` is being run
```

Comment:
fix comments

Reply:
Uh anything in particular you want me to fix?